Fix the log levels mapping (#597)
ansible/pylibssh: docs/user_guide.rst

   :dedent: 4

Logging events
==============

The |project| is passing all the ``libssh`` log messages into the Python
:mod:`logging` subsystem by default. This is initialized when new
session is created using ``Session`` constructor.

You can configure your loggers and handlers as in any other Python

> project. The |project| Logger name is ``ansible-pylibssh``.

Reviewer: Let's mark it as
```suggestion
project. The |project| Logger name is ``'ansible-pylibssh'``.
```
since it's a string literal.

.. literalinclude:: _samples/logging.py
   :language: python
   :start-at: ssh = Session()
   :end-before: # Set loglevel on session
   :emphasize-lines: 1

.. attention::

    When performance is critical, you can use the
    ``Session.set_log_level(ANSIBLE_PYLIBSSH_NOLOG)``

>     API to prevent libssh emitting the log messages at source.

Reviewer: I foresee this being a point of confusion so let's try to prevent common mistakes:
```suggestion
    API to prevent libssh emitting the log messages at source.
    Note that it is different from :data:`logging.NOTSET` which has
    the opposite semantics.
```

Author: https://docs.python.org/3/library/logging.html#logging.NOTSET but otherwise ack.

   :dedent: 4

Logging events
==============

The |project| is passing all the ``libssh`` log messages into the Python

> :mod:`python:logging` subsystem by default. This is initialized when new

Reviewer: ```suggestion
:external+python:mod:`logging` subsystem by default. This is initialized when new
```

Author: Replaced in all the places to this linking

> session is created using ``Session`` constructor.

Reviewer: ```suggestion
session is created during the ``Session`` initialization.
```

You can configure your loggers and handlers as in any other Python
project. The |project| Logger name is ``'ansible-pylibssh'``.

.. literalinclude:: _samples/logging.py
   :language: python
   :start-at: ssh = Session()
   :end-before: # Set log level on session
   :emphasize-lines: 1

.. attention::

    When performance is critical, you can use the
    ``Session.set_log_level(ANSIBLE_PYLIBSSH_NOLOG)``

>     API to prevent libssh emitting the log messages at source.

Reviewer: ```suggestion
    API to prevent ``libssh`` from emitting the log messages at the source.
```

>     API to prevent ``libssh`` emitting the log messages at the source.

Reviewer: "prevent X *from* doing smth"
```suggestion
    API to prevent ``libssh`` from emitting the log messages at the source.
```
